### PR TITLE
Allow blank content on notebook revision creation (fixes #2445)

### LINKED ISSUE #2445: Get an error when completely erasing default notebook content
How to reproduce:

1. Create a new notebook
2. Erase all content
3. Switch tabs or wait a while

Result:

Get this error:

image

It appears that the `/api/notebooks/<notebook>/revisions/` endpoint does not accept an empty string, but it really should. Code which should be changed probably around here: https://github.com/iodide-project/iodide/blob/master/server/notebooks/api_views.py

### PR DESCRIPTION
**Pull request by imranariffin**
### Pull Request checklist

- [x] **Documentation**: If this feature has or requires documentation, the relevant docs have been updated.
- [x] **Changelog**: This PR updates the [changelog](https://github.com/iodide-project/iodide/blob/master/CHANGELOG.md) with any user-visible changes.
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not

This PR should fix #2445 by allowing blank value on `"content"` field of the `POST /notebook/<id>/revision/` endpoint.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 - Vertically align text in language selector(fixes #2446)
 - Allow running simple-serve from a port other than 8000 (fixes #2444)
+- Fix bug unable to create revision with blank content (#2445)
```
Comment (wlach) [inline CHANGELOG.md]:
```suggestion
- Allow blank content on notebook revision creation (fixes #2445) (#2455)
```

**Comment by wlach** [comment]
@imranariffin Actually looks like I don't have permission to push to your repository. Could you apply my suggestion? Happy to merge afterwards.

**Comment by imranariffin** [comment]
> @imranariffin Actually looks like I don't have permission to push to your repository. Could you apply my suggestion? Happy to merge afterwards.

Done the requested change, thank you for the review!

**Comment by wlach** [comment]
Thanks so much!

**Comment by imranariffin** [comment]
Thank you! Here goes my first (significant) open source PR yay!

**Pull request merged**